tlatools/org.lamport.tlatools/pom.xml

```diff
@@ -61,7 +61,7 @@
         <plugin>
 	    <groupId>org.apache.maven.plugins</groupId>
 	    <artifactId>maven-antrun-plugin</artifactId>
-	    <version>1.8</version>
+	    <version>3.1.0</version>
```

Comment:
Isolate this change?!

Reply:
This is necessary because older versions of the maven-antrun-plugin use an older version of ant that doesn't have the `release` field in the `javac` target